metrics/thresholds_test.go

```diff
@@ -638,12 +638,18 @@ func TestThresholdsRunAll(t *testing.T) {
 	}
 }
 
-func TestThresholds_Run(t *testing.T) {
+func TestThresholdsRun(t *testing.T) {
 	t.Parallel()
 
+	filledTrendSink := &TrendSink{}
+	filledTrendSink.Add(Sample{Time: time.Now(), Value: 70})
+	filledTrendSink.Add(Sample{Time: time.Now(), Value: 80})
+	filledTrendSink.Add(Sample{Time: time.Now(), Value: 90})
```

Comment:
This needs to be in the `t.Run` below so that you don't wokr on the same Trend - this is where the race comes from

Reply:
Thanks 🙇🏻